python/activator/middleware_interface.py

```diff
@@ -750,18 +750,17 @@ def export_outputs(self, visit: Visit, exposure_ids: set[int]) -> None:
         # TODO: this method will not be responsible for raws after DM-36051.
         self._export_subset(visit, exposure_ids, "raw",
                             in_collections=self.instrument.makeDefaultRawIngestRunName(),
-                            out_collection=None)
+                            )
 
-        umbrella = self.instrument.makeCollectionName("prompt-results")
```

Comment:
In `export_outputs`'s  documentation some lines above, it still says 
> ... the outputs can be found in ``"<instrument>/prompt-results" 

and needs to be updated.